src/tests/dictionaryConcepts/container/EditConcept.test.jsx

```diff
@@ -625,4 +652,15 @@ describe('Test suite for mappings on existing concepts', () => {
     instance.updateSourceEventListener(event, url);
     expect(instance.state.mappings[1].map_type).toEqual('SAME-AS');
   });
+
+  it('should call removeCurrentAnswer function to remove mapping', () => {
+    const answerUrl = 'url';
+    const frontEndUniqueKey = 'unique';
+    const answer = {};
+    const info = { answerUrl, frontEndUniqueKey, answer };
+    const instance = wrapper.find('EditConcept').instance();
+    const spy = jest.spyOn(instance, 'removeCurrentAnswer');
+    instance.removeCurrentAnswer(info);
```

Comment:
Are you testing for a method call that you are directly making here?

Reply:
Updated